src/browser/components/DestructiveConfirmModal.jsx

```diff
@@ -13,9 +13,14 @@ export default function DestructiveConfirmationModal(props) {
     cancelLabel,
     onAccept,
     onCancel,
+    modalContainer,
     ...rest} = props;
   return (
-    <Modal {...rest}>
+    <Modal
+      container={modalContainer}
+      restoreFocus={false}
+      {...rest}
+    >
```

Comment:
Missing from https://github.com/mattermost/desktop/pull/1056.
Causing React error when this modal appears.